browser/ui/views/omnibox/brave_omnibox_view_views.cc

```diff
@@ -0,0 +1,65 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/omnibox/brave_omnibox_view_views.h"
+
+#include "base/feature_list.h"
+#include "brave/browser/ui/toolbar/brave_location_bar_model_delegate.h"
+#include "brave/common/url_constants.h"
+#include "components/omnibox/browser/omnibox_edit_model.h"
+#include "content/public/common/url_constants.h"
+#include "ui/base/clipboard/scoped_clipboard_writer.h"
+#include "url/gurl.h"
+
+namespace {
+// Copied from omnibox_view_views.cc
+constexpr base::Feature kOmniboxCanCopyHyperlinksToClipboard{
```

Comment:
why are we copying this? It's completely pointless because we'd have to track upstream changes anyway which is a problem

Reply:
done.